src/shared/renderUtils.jsx

```diff
@@ -1,19 +1,60 @@
 import React from "react";
 
-function renderBody(body){
+function renderBody(body, scope = "", className = ""){
 	const paragraphs = body.split("\n"); // splits the string at every line-break
-	return paragraphs.map(renderParagraph);
+
+	return paragraphs.map(
+		renderParagraph.bind(this, scope, className)
+	);
 }// renderBody
 
-function renderParagraph(text, index){
-	const key = `about__body__paragraph_${index}`;
+function renderParagraph(scope, className, text, index){
+	const key = `${scope}__body__paragraph_${index}`;
+	
+	//if there is a link inside the text, cut it out
+	const link = findLink(text) || {};
+	if(link) text = text.slice(0, link.index);
+
 	return (
-		<p key={ key }>
+		<p 
+			className={ className }
+			key={ key }
+		>
 			{ text }
+			
+			{ link && (
+				<a href={ link.destination } >
+					{ link.text }
+				</a>
+			)}
 		</p>
 	);
 }// renderParagraph
 
+function findLink(text){
+	const linkLabelStart = text.indexOf("[");
+
+	if(linkLabelStart === -1){
```

Comment:
```suggestion
	if(labelStart === -1){
```

Reply:
Also, I'm going to leave this in here as it's personal preference, but you gotta choose here what format you're going to do throughout.  Either:
```
if(failCondition) return;

...do stuff...
```
or
```
if(passCondition){
    ...do stuff...
} else return;
```

The first one is called an 'early return' or an 'escape return' and it tends to be used by peeps who dislike indentation and so by returning early they don't need an `else`, and so the main body of the function can stay at level 1 indentation; whereas the latter is better for A/B comparissons and code-folding.

Either works, but this one here is a mishmash of both - an early return _and_ an `else` statement.  But yeah - whichever you choose, make sure you're consistant with it throughout your codebase.

Reply:
Cool, ty for the feedback on this.

I like both but prefer the simplicity of the early return. Updated.